aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -213,6 +213,10 @@ Sensitive files exist on the app's physical path, such as *\<assembly>.runtimeco
 
 **The *web.config* file must be present in the deployment at all times, correctly named, and able to configure the site for normal start up. Never remove the *web.config* file from a production deployment.**
 
+### web.config transformations
+
+If you require *web.config* transformations (for example, setting environment variables based on the configuration, profile, or environment), see <xref:host-and-deploy/webconfig-transformation>.
```

Comment:
"need" would be less formal than "require"

Reply:
:+1: Sure, I only truly avoid "need" when dealing with non-human life forms ... .NET Core 👽 :smile:.